sci-biology/bwa/bwa-0.7.12.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit base toolchain-funcs
+
+DESCRIPTION="Burrows-Wheeler Alignment Tool, a fast short genomic sequence aligner"
+HOMEPAGE="http://bio-bwa.sourceforge.net/"
+SRC_URI="mirror://sourceforge/bio-bwa/${P}.tar.bz2"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE=""
+KEYWORDS="~amd64 ~x86 ~x64-macos"
+
+PATCHES=( "${FILESDIR}"/${PN}_Makefile.patch )
+
+src_compile() {
+	tc-export CC AR
+	default
+}
```

Comment:
drop this phase and use src_prepare instead.

   src_prepare() {
       epatch ${PATCHES[@]}
       tc-export CC AR
   }